### ISSUE TITLE
Escape `{` and `}` in reports

### ISSUE THREAD

**Issue by jsyiek** [jsyiek/AU2#19]
![image](https://github.com/jsyiek/AU2/assets/43001551/e9b98561-4681-47a8-bfc0-5a1ed6ab6732)

**Comment by jsyiek** [comment]
format strings don't play well with `{` in reports

**Comment by jsyiek** [comment]
could be fixed by `.replace`'ing the `{` or `}` with their html code equivalents? the issue is that all pseudonyms/heads/news are escaped (which would kill the escape), so it's not a complete solution

**Comment by NerdyNinja11** [comment]
'If you need to include a brace character in the literal text, it can be escaped by doubling: {{ and }}' - python docs.
Would this work as initial sanitisation to reports/headlines?

**Comment by jsyiek** [comment]
> 'If you need to include a brace character in the literal text, it can be escaped by doubling: {{ and }}' - python docs.
> 
> Would this work as initial sanitisation to reports/headlines?

Probably! All text inputs should sanitize in the front-end with a .replace call.

**Comment by thenewtonium** [comment]
> 'If you need to include a brace character in the literal text, it can be escaped by doubling: {{ and }}' - python docs. Would this work as initial sanitisation to reports/headlines?

Does this not cause issues with page generation though? Since there the relevant strings do not get passed through `.format` and so will be rendered with double rather than single braces.

Since it seems to only be `inquirer` the thas an issue with curly braces, might it not make more sense to escape strings being passed into this instead? PR #51 demonstrates what I mean.

**Comment by jsyiek** [comment]
Yeah, this is what I was saying about sanitizing in the front-end only.
